### PR TITLE
[14.0][IMP] delivery_procurement_group_carrier: Set Carrier on new picking from procurement.group

### PR DESCRIPTION
**Pull request by mt-software-de**
If run a procurement where the carrier is set.
I would be helpful, if the carrier on a new create picking is automatically set.

### CODE REVIEW

**Comment by mt-software-de** [comment]
@jbaudoux could you do the review?

File: delivery_procurement_group_carrier/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _get_new_picking_values(self):
+        values = super()._get_new_picking_values()
+        carrier = self.group_id.carrier_id
+        if carrier and self.picking_type_id.code in ["outgoing", "incoming"]:
+            values["carrier_id"] = carrier.id
```
Comment (rousseldenis) [inline delivery_procurement_group_carrier/models/stock_move.py:12]: @mt-software-de Is it a real behavior you want? As Odoo behavior will be weird as it consider it should be only set on delivery pickings.

Shouldn't you use this instead ? https://github.com/OCA/delivery-carrier/blob/14.0/stock_picking_delivery_link/models/stock_picking.py#L24

Comment (mt-software-de) [reply delivery_procurement_group_carrier/models/stock_move.py:12]: Yes it is a real behavior, we need for return pickings from a customer to our warehouse.

**Comment by mt-software-de** [comment]
ping @jbaudoux

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**